lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1099,7 +1124,7 @@ def test_create_order_success(self):
             photo_id_image=self.IMAGE_DATA
         )
         json_response = json.loads(response.content)
-        self.assertIsNotNone(json_response.get('orderNumber'))
+        self.assertIsNotNone(json_response['payment_form_data'].get('orderNumber'))  # TODO not canonical
```

Comment:
this test runs against the shoppingcart backend and assumes the cybersource processor, which has 'orderNumber' in its expected form data params.  I've left it alone except to correct the location in the response payload where the key is expected to be found.